openapi.yaml

```diff
@@ -273,25 +296,37 @@ paths:
     get:
       summary: List all reviews
       description: Get a list of all reviews or filter by query parameters
+      operationId: getReviews
       tags:
-        - Reviews
+        - reviews
       parameters:
+        - name: gameId
+          in: query
+          type: string
+          required: false
+          description: Filter by the Id of the game
         - name: reviewer
           in: query
           type: string
           required: false
           description: Filter by the name of the game reviewer
-        - name: score_min
+        - name: scoreMin
           in: query
           type: number
           format: float
           required: false
+          default: 1
+          minimum: 1
+          maximum: 5
           description: Filter results by minimum review score
-        - name: score_max
+        - name: scoreMax
           in: query
           type: number
           format: float
           required: false
+          default: 5
+          minimum: 1
+          maximum: 5
```

Comment:
`scoreMin` and `scoreMax` filters here are repeated with the parameters in `GET /games`. I'd suggest to define it in the parameters section and make it reusable.